src/live-programming.rst

```diff
@@ -36,8 +36,132 @@ Direct Manipulation Programming
 Graphical User Interface Design
 -------------------------------
 
-Game Development
-----------------
+User Interfaces for Game Development
+====================================
+.. :Authors: - Lei Zhang
+
+Overview
+--------
+Starting with Spacewar in 1961, computer games have been around for over four and a half decades.
+Computer games are becoming increasingly important in lots of industries including software development, entertainment, education, military, and medicine.
+Game development itself has been considered as a combination of multiple disciplines including programming, design, art, modeling, animation, music, etc.
+
+.. container:: bib-item
+
+  .. bibliography:: game-development.bib
+    :filter: key == 'blow2004game'
+
+  This article presents a broad summary of the reasons why game development is hard nowadays.
+  In a high level, the author classified the difficulties into two categories: problems due to overall project size and complexity and problems due to highly domain-specific requirements.
+  Problems due to overall project size and complexity include the lack of excellent development tools, the inefficiency of development workflow, the barriers for multi-platform development, and the lack of excellent third-party components/engines.
+  Problems due to highly domain-specific requirements include the domain-specific knowledge about programming, mathematics, algorithms, computer graphics, etc.
+
+.. container:: bib-item
+
+  .. bibliography:: game-development.bib
+    :filter: key == 'washburn2016went'
+
+  This paper presents an analysis of 155 postmortems published on the gaming site Gamasutra.com. 
+  The paper identifies characteristics of game development, link the characteristics to positive and negative experiences in the postmortems and distill a set of best practices and pitfalls for game development.
+  Based on the analysis, the authors made the following recommendations to game developers: First, be sure to practice good risk management techniques;
+  Second, prescribe to an iterative development process and utilize prototypes as a method of proving features and concepts before committing them to the design;
+  Third, do not be overly ambitious in the design.
+
+As we can see in the above paper, there is a huge problem space in the game development domain including product (e.g. art, creativity, game design), development (e.g. tools, development process, team), resources (budge, hardware, publishers), and customer facing (e.g. marketing, feedback).
+In this compendium, our scope covers only a tiny amount of the whole problem space (i.e. **tools** and **languages** for game development).
```

Comment:
"In this compendium, our scope" -> "This compendium"
"tiny amount of the whole problem space (i.e. " -> ""